### PR TITLE
 feat: add support for Kubernetes v1.17.15

### PR DESCRIPTION
**Pull request by mboersma**
**Reason for Change**:

See https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.17.md#changelog-since-v11714

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [x] No
- [ ] Yes

**Requirements**:

- [x] Kubernetes artifacts built and pushed by Azure Pipelines
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by mboersma** [comment]
/hold

This contains a temporary commit for testing that must be discarded before merging.

**Comment by jackfrancis** [comment]
1.17.15 is failing due to this bug: https://github.com/kubernetes/kubernetes/issues/97178

tl;dr 1.17.15 was built w/ a 1.17.16-rc.0 tag

**Comment by mboersma** [comment]
> To mitigate this issue, a [fix](https://github.com/kubernetes/release/pull/1826) has been made to the logic that resulted in the official release and RC tags being created on the same commit, and we are planning on sending out patch releases for all versions tomorrow morning (12/18).
